srplasticity/tools.py

```diff
@@ -87,4 +87,6 @@ def __init__(self, func, args, **kwargs):
         self.kwargs = kwargs
 
     def __call__(self, x):
-        return self.minimizer(self.func, x0=x, args=self.args, **self.kwargs)
+        with warnings.catch_warnings():
```

Comment:
Why are you suppressing this warning?

Reply:
One of the reviewers said that if it isn't a source of concern, the warning should be suppressed because otherwise, the user might think something is wrong while running fit_srp_gridsearch

Reply:
It was a source of concern, I patched the cause of the error (see response to reviewers doc). Don't suppress the warnings.

Reply:
I removed the change

Reply:
But I am getting scipy/optimize/_numdiff.py:592: RuntimeWarning: invalid value encountered in subtract
  df = fun(x1) - f0